modules/datadict/php/datadict.class.inc

```diff
@@ -108,72 +108,12 @@ class Datadict extends \NDB_Menu_Filter
             )
         );
         $this->addBasicText('keyword', 'Search keyword');
-    }
-    /**
-    * Set Data Table Rows
-    *
-    * @param string $count the count of table rows
-    *
-    * @note   Set Data Table Rows
-    * @return bool
-    * @access private
-    */
-    function _setDataTableRows($count)
-    {
-
-        $x = 0;
 
-        foreach ($this->list as $item) {
-            $this->tpl_data['items'][$x][0]['value'] = $x + $count;
-            $i = 1;
-
-            foreach ($item as $key => $val) {
-                if ($key =='description') {
-                    $key = $item['Name'] . "___" . "description";
-                }
-                if ($key == "sourceFrom" && $val== "parameter_session") {
-                    $val = $item['Name'];
-                }
-                $this->tpl_data['items'][$x][$i]['name']  = $key;
-                $this->tpl_data['items'][$x][$i]['value'] = $val;
-
-                $i++;
-            }
-
-            $x++;
-        }
         return true;
     }
-    /**
```

Comment:
@xlecours not sure why this is considered dead ? the function is being used and it is different then the parent function

Reply:
okai after further tracking I realised that the additional if statement is no longer being used. so this is fine